### PR TITLE
Add a helper function in sei-js to calculate expected APR.

### PR DESCRIPTION
**Pull request by mj850**
Our APR calculation in sei-app is based on hardcoded values on token release. Instead, we want a live APR calculator that queries real data about the upcoming mint schedule.

APR calculation used here is essentially: `Upcoming tokens to be minted in next 365 days` / `bondedTokens`

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -65,7 +66,8 @@
 		"@babel/preset-env": "^7.22.20",
 		"@babel/preset-typescript": "^7.22.15",
 		"@types/elliptic": "^6.4.14",
-		"@types/sha.js": "^2.4.1"
+		"@types/sha.js": "^2.4.1",
+		"long5": "npm:long"
```
Comment (besated) [inline packages/core/package.json:70]: why is this aliased as `long5`?

File: packages/core/src/lib/utils/apr.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    return Math.round(endDate.diff(startDate, 'days', true)) + 1;
+}
+
+interface ReleaseSchedule {
+    startDate: Moment;
+    endDate: Moment;
+    tokenReleaseAmount: number;
+}
+
+function CreateReleaseSchedule(start_date: string, end_date: string, token_release_amount: Long): ReleaseSchedule {
```
Comment (besated) [inline packages/core/src/lib/utils/apr.ts:126]: can you make function names camelcase? so `createReleaseSchedule`

**Review by besated — CHANGES_REQUESTED** [review]

File: packages/core/package.json (hunk 1 above)
Comment (mj850) [reply packages/core/package.json:70]: Was running into some weird issues where we import 2 different types of Long using 'import Long from 'long';
There's [@types/long](https://www.npmjs.com/package/@types/long) that's used by one of the interfaces in cosmjs-types and [long.js](https://github.com/dcodeIO/Long.js/) that's used by the ScheduledTokenReleaseSDKType created by our proto library.

The 2 types don't seem to be substitutable for each other so I added an alias for long.js

File: packages/core/src/lib/utils/apr.ts (hunk 2 above)
Comment (mj850) [reply packages/core/src/lib/utils/apr.ts]: Fixed thanks!

**Review by besated — APPROVED** [review]

**Pull request merged**